docs/course_materials/exercises/01_Linux/check_class_exercise_1.py

```diff
@@ -0,0 +1,28 @@
+import os.path
+from pathlib import Path
+import check_helper
+
+home = str(Path.home())
```

Comment:
I think it could a global variable, maybe create a config file and add this variable so we won't have this twice in two places.

Reply:
Made the 'home' a global variable in check_helper, this file would be used for all the exercises.